pages/extensions/internal_links/pod_internal_link_rewriter.py

```diff
@@ -72,6 +72,13 @@ def rewrite_link(self, internal_link, query_and_anchor):
     return result
 
   def get_site_link(self, internal_link):
+    # Static files are located inside the pod which will make Grow
+    # create virtual docs when calling get_doc with such a path.
+    # This will make doc.exists equal True while doc.url.path only returns
+    # garbage. To work around this only further check for docs in '/content'
+    if internal_link.startswith('/static'):
```

Comment:
can we please extract these into a config file.

Reply:
Would putting this into a well-named constant already work for you? Struggling a little to find a either existing config file to add this to or create a new one where it wouldn't have any context.

Reply:
theoretically `platform/config/shared.json` would be a good place. Once we re-org the repo structure, I'd propose a root level config folder for these kinds of things.

Reply:
constant is fine for now though